nservicebus/azure-storage-queues/multi-storageaccount-support_aliases_asq_[7,].partial.md

```diff
@@ -0,0 +1,24 @@
+## Using aliases for connection strings to storage accounts for Scale Out
+
+In order to prevent accidentally leaking connection string values, it is recommended to use aliases instead of raw connection strings. When applied, raw connection string values are replaced with registered aliases removing the possibility of leaking a connection string value. The concept of using aliases for connection strings to storage accounts has been introduced in Version 7. When using a single account, aliasing connection string is limited to just enabling it. When Scaling Out is applied, it requires registering an alias for each storage account.
```

Comment:
It's on the multi-storage-account page and discusses how to scale with many accounts, to overcome quotas of a single account. Should it be described as `Scale Up` then? This covers your scenario as well. Many endpoints on one acccount in Scale Out mode can easily saturate its throughput. Having this said, I see current wording as a valid header, covering all these cases.